flocker/control/httpapi.py

```diff
@@ -796,6 +799,42 @@ def delete_container_configuration(self, name):
         # Didn't find the application:
         raise CONTAINER_NOT_FOUND
 
+    @app.route("/configuration/_compose", methods=['POST'])
+    @user_documentation(
+        """
+        Private API endpoint used by flocker-deploy.
```

Comment:
Maybe a little more emphasis than this.  What does "private" mean to the reader of our API documentation, anyway?  "... temporarily used by ...", "This will be removed soon.", that sort of thing.